check-tree-sitter-crates.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+# Stop at the first error
+set -e
+
+# Get all tree-sitter crates from the analyzed branch Cargo.toml
+TS_CRATES=`grep "tree-sitter-*" Cargo.toml | tr -d ' '`
```

Comment:
This won't include the `tree-sitter` dependency, right?

Reply:
Yes, it includes the `tree-sitter` dependency